### PR TITLE
[civ2][docker/3] wanda for ray.py38.cu118.base image

### PR DESCRIPTION
**Pull request by can-anyscale**
Wanda for building raybase image. Hard code python version and platform for now. 

Test:
- CI

### CODE REVIEW

File: ci/docker/ray.py38.cu118.base.wanda.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: "raypy38cu118base"
+froms: ["nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04"]
+dockerfile: docker/base-deps/Dockerfile
+srcs: []
```
Comment (aslonnie) [inline ci/docker/ray.py38.cu118.base.wanda.yaml:4]: if the `src` list is empty, you can just drop this line I think.

File: .buildkite/_forge.rayci.yml [hunk 2]
```diff
@@ -3,8 +3,15 @@ steps:
   - name: forge
     wanda: ci/docker/forge.wanda.yaml
 
+  # Images for building executable and docker images
+
   - name: manylinux
     wanda: ci/docker/manylinux.wanda.yaml
+
+  - name: raypy38cu118base
+    wanda: ci/docker/ray.py38.cu118.base.wanda.yaml
```
[the remark this answers is not in the document]
Comment (aslonnie) [reply .buildkite/_forge.rayci.yml:10]: yeah.. I am still a bit worry about all the combinations..

maybe we need to add matrix support for rayci and wanda ..

**Review by aslonnie — APPROVED** [review]

File: .buildkite/_forge.rayci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (aslonnie) [reply .buildkite/_forge.rayci.yml:10]: should we add a non cuda / cpu image first? or at least also add the non-cuda one at the same time here?

[the remark this answers is not in the document]
Comment (can-anyscale) [reply .buildkite/_forge.rayci.yml:10]: i was hoping to add matrix feature to wanda first; of if it's not that trivial maybe I will enumerate everything first and matrix them later

**Pull request merged**